command/resource/helper.go

```diff
@@ -135,6 +136,24 @@ func isHCL(v []byte) bool {
 	return true
 }
 
+func ParseResourceInput(filePath string, stdin io.Reader) (*pbresource.Resource, error) {
```

Comment:
This needs the same `isHCL` patch from `ParseResourceFromFile` that I recently added.

Reply:
Ahhh.. yes.